README.md

```diff
@@ -70,6 +70,15 @@ pub fn main() !void {
     try surface.fillRect(null, surface.mapRgb(128, 30, 255));
     try window.updateSurface();
 
```

Comment:
These modifications do get the window to show up for me, but the window is not painted purple as originally intended by my code. However, this is a problem with my code and not your changes. I'm not sure if you wanted to try fixing this bug in this PR, if not then disregard this comment.